base/db/tests/Rcheck_reference.log

```diff
@@ -38,53 +38,21 @@
 * checking replacement functions ... OK
 * checking foreign function calls ... OK
 * checking R code for possible problems ... NOTE
-clone_pft: no visible binding for global variable ‘name’
-clone_pft: no visible binding for global variable ‘id’
-clone_pft: no visible binding for global variable ‘created_at’
-clone_pft: no visible binding for global variable ‘updated_at’
-clone_pft: no visible binding for global variable ‘pft_id’
+
+
 db.exists: no visible binding for '<<-' assignment to ‘user.permission’
 db.exists: no visible binding for global variable ‘user.permission’
-dbfile.check: no visible binding for global variable ‘container_type’
-dbfile.check: no visible binding for global variable ‘container_id’
-dbfile.check: no visible binding for global variable ‘machine_id’
-dbfile.check: no visible binding for global variable ‘updated_at’
-dbHostInfo: no visible binding for global variable ‘sync_host_id’
-get_run_ids: no visible binding for global variable ‘run_id’
-get_users: no visible binding for global variable ‘id’
-get_workflow_ids: no visible binding for global variable ‘workflow_id’
-get.trait.data.pft: no visible binding for global variable ‘pft_id’
-get.trait.data.pft: no visible binding for global variable ‘created_at’
-get.trait.data.pft: no visible binding for global variable ‘stat’
-get.trait.data.pft: no visible binding for global variable ‘trait’
-insert.format.vars: no visible binding for global variable ‘id’
-insert.format.vars: no visible binding for global variable ‘name’
-insert.format.vars: no visible global function definition for ‘collect’
+
 load_data_single_run: no visible global function definition for
   ‘read.output’
 load_data_single_run: no visible binding for global variable ‘var_name’
 load_data_single_run: no visible binding for global variable ‘vals’
-load_data_single_run: no visible binding for global variable ‘posix’
 match_dbcols: no visible global function definition for ‘head’
 match_dbcols: no visible binding for global variable ‘.’
 match_dbcols: no visible binding for global variable ‘as’
-query_pfts: no visible binding for global variable ‘name’
```

Comment:
@moki1202 The current CI failure is because this warning hasn't been fixed yet. Confusingly, we have files named both `query.pft.R` (which you have cleaned up in this PR) and `query_pfts.R` (which you haven't yet modified). To fix the build error you can either do the updates in `query_pfts.R`, or re-add this line to `Rcheck_reference.log` and save the cleanup in `query_pfts` for another PR -- either way is OK with me.

Reply:
ill fix this file as soon as my classes are over  @infotroph

Reply:
@infotroph just updated query_pfts.R, turns out I did forget to update this file. I apologize.